sys/include/net/lora.h

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (C) 2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_lora LoRa modulation
+ * @ingroup     net
+ * @brief       LoRa modulation header definitions
+ * @{
+ *
+ * @file
+ * @brief       LoRa modulation header definitions
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ */
+
+#ifndef NET_LORA_H
+#define NET_LORA_H
+
+#include <stdint.h>
+#include <stdlib.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    LoRa modulation default values
+ * @{
+ */
+#define LORA_FREQUENCY_RESOLUTION      (61.03515625)     /**< Frequency resolution in Hz */
```

Comment:
Since those values are default, maybe I should rename them to *_DEFAULT ?

Reply:
+1

Reply:
updated